src/modules/ekf2/EKF/ekf.h

```diff
@@ -521,6 +521,18 @@ class Ekf final : public EstimatorInterface
 	bool resetGlobalPosToExternalObservation(double lat_deg, double lon_deg, float accuracy,
 			uint64_t timestamp_observation);
 
+	/**
+	* @brief Resets the wind states to an external observation
+	*
+	* @param wind_speed The wind speed in m/s
+	* @param wind_direction The azimuth (from true north) from where the wind is flowing from in degrees (0 to 360)
```

Comment:
This might be the mavlink convention, but it's now how we use it in EKF2. We should convert it to a "wind direction vector" (when receiving the message in ekf2.cpp) before sending it to the EKF backend